Description: Change - by \- to use minus sign instead hyphen in manpage.
Author Mònica Ramírez Arceda <monica@probeta.net>
Last-Update: 2011-02-13

--- a/splitvt.1
+++ b/splitvt.1
@@ -11,52 +11,52 @@
 .LP
 .TP
 .TB
-.I -upper command
+.I \-upper command
 Runs 'command' in the upper window
 .TP
 .TB
-.I -lower command
+.I \-lower command
 Runs 'command' in the lower window
 .TP
 .TB
-.I -bottom
+.I \-bottom
 Stats with the focus in the bottom window
 .TP
 .TB
-.I -s numlines
+.I \-s numlines
 Sets 'numlines' to the number of lines in the top window.  This number
 will be modified if the screen isn't big enough to handle the full
 size.
 .TP
 .TB
-.I -t title
+.I \-t title
 Sets an xterm title bar to 'title'
 .TP
 .TB
-.I -login
+.I \-login
 Runs the programs under each window as though they were login shells.
 This is useful to have csh source it's .login file, bash source .profile,
 etc.
 .TP
 .TB
-.I -nologin
+.I \-nologin
 Ovverrides any "set login on" statement in the startup file.
 .TP
 .TB
-.I -rcfile file
+.I \-rcfile file
 Loads 'file' as the startup file instead of ~/.splitvtrc.
 .TP
 .TB
-.I -norc
+.I \-norc
 Suppresses loading your ~/.splitvtrc
 .TP
 .TB
-.I -f
+.I \-f
 Allow setting the height of either the top or the bottom window to less than
 3 lines.
 .TP
 .TB
-.I -v
+.I \-v
 Print out the version number of splitvt
 .LP
 .SH DESCRIPTION
@@ -85,7 +85,7 @@
 .LP
 .TP
 .TB
-.I '-'
+.I '\-'
 Shrink the current window
 .LP
 .TP
@@ -125,7 +125,7 @@
 .LP
 More functions may be added in the future.
 
-If the -s command line option is used, the upper window will be bound 
+If the \-s command line option is used, the upper window will be bound 
 to the user defined number of lines even if the screen is resized.  
 Otherwise, the screen will be split into two equal parts based on 
 the new size of the screen.
@@ -156,8 +156,8 @@
 set quote_char          ^V
 set switch_char         ^W
 set upper_lines         12
-run -upper              /bin/sh
-run -lower              top
+run \-upper              /bin/sh
+run \-lower              top
 set login               on
 # This next line would override the above run statements
 #run /bin/tcsh
@@ -198,7 +198,7 @@
 weaknesses splitvt might posess.
 .SH BUGS
 When used with a certain public domain version of ksh, you have to
-manually kill -9 on the shell processes after you quit splitvt.  This
+manually kill \-9 on the shell processes after you quit splitvt.  This
 has to do with a bug in that version of ksh (Version 06/03/86a and
 possibly others).
 
